### ISSUE TITLE
GCal Events or Agenda module?

### ISSUE THREAD

**Issue by Graham-72** [backdrop-ops/contrib#115]
I need a module that will display, as a block, a list of upcoming events. Drupal has two (or more) contributed modules that would apparently do this, and I am wondering whether anyone has a view on which might be better?

I have gone ahead and ported 'Agenda' and that seems to be working OK in Backdrop. 
See https://github.com/backdrop-contrib/agenda

I will now have a quick look at GCal Events.

**Comment by klonos** [comment]
Thanx @Graham-72 for the port. Couldn't you achieve the same using views? ...or is this about pulling events from google calendar?

Also, this is the second time I see you forget/miss to add a description and website for a repo of a contrib module you port:

![backdrop-contrib_repo-description_and_website_empty](https://cloud.githubusercontent.com/assets/2423362/10693623/294b4816-79e5-11e5-93d0-1479e01f94d4.png)

I went ahead and did that for you (again):

![backdrop-contrib_repo-description_and_website](https://cloud.githubusercontent.com/assets/2423362/10693634/3bc4f4d8-79e5-11e5-9f4f-d4688452c2b2.png)

Hope it's ok :wink:

**Comment by Graham-72** [comment]
Thanks @klonos for correcting my omission. I am sorry to have put you to this trouble and I promise to try and do better!

Yes, this is about pulling events from Google Calendar. What would be really nice perhaps would be to import each one as an entity in Backdrop, then use Views to create listings, tables etc

**Comment by Graham-72** [comment]
@klonos  I now realise that there are two more repos I have very recently transferred to backdrop-contrib without filling in the description field, and now they are there I cannot see a way of correcting this myself.

They are (with my suggestion for short descriptions):
GCal -  Provides APIs for managing Google calendars.
GAuth - Allows for authentication with Google enabling use of Google APIs

Your assistance once again will be much appreciated. Thanks.

**Comment by klonos** [comment]
> ...I am sorry to have put you to this trouble and I promise to try and do better!

No, no. Perhaps because English is not my main language (plus sleep deprivation plays a part), sometimes I get the wrong message across. I simply wanted to point a "flaw" in your workflow of creating contrib module repos that I noticed you were being consistent with.

> Yes, this is about pulling events from Google Calendar. What would be really nice perhaps would be to import each one as an entity in Backdrop, then use Views to create listings, tables etc

Exactly what I had in mind.

**Comment by klonos** [comment]
> I now realise that there are two more repos I have very recently transferred to backdrop-contrib without filling in the description field, and now they are there I cannot see a way of correcting this myself.

Perhaps a permission issue with the contrib authors group? @quicksketch??

> Your assistance once again will be much appreciated. Thanks.

Done! I've added only the descriptions. Once they get official releases and hence also a valid bacdropcms.org URL, just ping me and I'll add those too.

**Comment by klonos** [comment]
I'm positively sure that the fact that I belong to the contrib "Owners" team is what allows me to edit these details of the repos post-creation: https://github.com/orgs/backdrop-contrib/teams
